config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-periodics-main.yaml

```diff
@@ -1,132 +1,148 @@
-    testgrid-tab-name: capg-conformance-main-ci-artifacts
-    testgrid-alert-email: sig-cluster-lifecycle-cluster-api-gcp-alerts@kubernetes.io, release-team@kubernetes.io
-    testgrid-num-failures-to-alert: "2"
+  - name: periodic-cluster-api-provider-gcp-build-main
+    interval: 4h
+    decorate: true
+    cluster: eks-prow-build-cluster
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: cluster-api-provider-gcp
+        base_ref: main
+        path_alias: "sigs.k8s.io/cluster-api-provider-gcp"
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230406-23cb1879e3-master
+          command:
+            - "runner.sh"
+            - "./scripts/ci-build.sh"
+          resources:
+            limits:
+              cpu: 4
+              memory: 6Gi
+            requests:
+              cpu: 4
+              memory: 6Gi
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-gcp
+      testgrid-tab-name: capg-build-main
+      testgrid-alert-email: sig-cluster-lifecycle-cluster-api-gcp-alerts@kubernetes.io
+      testgrid-num-failures-to-alert: "2"
+  - name: periodic-cluster-api-provider-gcp-test-main
+    interval: 4h
+    decorate: true
+    cluster: eks-prow-build-cluster
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: cluster-api-provider-gcp
+        base_ref: main
+        path_alias: "sigs.k8s.io/cluster-api-provider-gcp"
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230406-23cb1879e3-master
+          args:
+            - "runner.sh"
+            - "./scripts/ci-test.sh"
+          resources:
+            limits:
+              cpu: 4
+              memory: 6Gi
+            requests:
+              cpu: 4
+              memory: 6Gi
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-gcp
+      testgrid-tab-name: capg-test-main
+      testgrid-alert-email: sig-cluster-lifecycle-cluster-api-gcp-alerts@kubernetes.io
+      testgrid-num-failures-to-alert: "2"
+  - name: periodic-cluster-api-provider-gcp-make-conformance-main
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    decorate: true
+    decoration_config:
+      timeout: 3h
+    interval: 12h
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: cluster-api-provider-gcp
+        base_ref: main
+        path_alias: "sigs.k8s.io/cluster-api-provider-gcp"
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230406-23cb1879e3-master
+          env:
+            - name: "BOSKOS_HOST"
```

Comment:
I don't think we have boskos with a GCE book in the EKS cluster, this job either doesn't need to set BOSKOS_HOST or actually does depend on external GCP assets.

given "script/ci-conformance.sh", I'd guess the latter.

Reply:
This job is not being moved to the aws cluster, just the build and unit test

Reply:
✨ git diff ✨